sormas-app/app/src/main/java/de/symeda/sormas/app/backend/facility/FacilityDao.java

```diff
@@ -145,6 +149,18 @@ public Date getLatestChangeDateByRegion(Region region) {
 			where.and().eq(InfrastructureAdo.ARCHIVED, false);
 			where.and().eq(AbstractDomainObject.SNAPSHOT, false);
 			where.and().ne(Facility.UUID, FacilityDto.OTHER_LABORATORY_UUID).query();
+
+			User currentUser = ConfigProvider.getUser();
+			UserRole.getJurisdictionLevel(currentUser.getUserRoles());
+
+			if (currentUser.getCommunity() != null) {
```

Comment:
I'm pretty sure that this code is problematic. Laboratories usually don't have a district or community, and app users normally are on district, community or facility level, so this will almost always return an empty list and lead to null pointers or empty mandatory fields. Do we really need this logic? If the user is allowed to edit the laboratory field, they should also be allowed to see the names of all laboratories; if they don't, maybe we actually have to do the pseudonymization in the user interface. The alternative (which I would actually prefer right now) would be to get rid of the pseudonymization of laboratory names for now because I don't really think it's as important. So maybe just remove this code for now and we'll revisit it later.